### DISCUSSION TITLE
2022/android-language/

### DISCUSSION THREAD

**Discussion by giscus** [xzsk2/xzsk2.github.io#5] [Announcements]
# 2022/android-language/

使用 ADB 开启 Lawnchair 应用抽屉的中英混排 无需root

https://sakari.top/2022/android-language/

**Comment by Goooler** [comment]
挺有用的，现在 Lawnchair 在不开启上述设置的情况下也支持中英文混排了，见 https://github.com/LawnchairLauncher/lawnchair/issues/2986#issuecomment-1617549516
